support/fixtures/factories/feeds.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('feed')
+  .option('fromServer', false)
+  .sequence('id')
+  .attrs({
+    createdAt: () => faker.date.past().toISOString(),
+    criticalThreshold: () => faker.random.number({ min: 0, max: 5 }),
+    degradedThreshold: () => faker.random.number({ min: 0, max: 5 }),
+    downAfter: () => faker.random.number({ min: 1000, max: 9999 }),
+    facilityId: () => faker.random.number({ min: 0, max: 40 }),
```

Comment:
For `facilityId` and `feedTypeId`, use the appropriate fixtures to get the ID instead of using a random number